mappings/net/minecraft/client/particle/ParticleLavaEmber.mapping

```diff
@@ -1,7 +1,7 @@
-CLASS none/bni net/minecraft/client/particle/ParticleLava
+CLASS none/bni net/minecraft/client/particle/ParticleLavaEmber
```

Comment:
Wouldn't mind going with it, though I lean towards `LavaEmber`. :+1: / :-1: on using `Ember`?